### PR TITLE
Add new app logo

### LINKED ISSUE #58: Add new app logo
Add the new app logo from https://www.dropbox.com/sh/rbty2an2zid6fla/AABkal72BP_dY3b7RF8m7TJra?dl=0
Replace logo placeholders

### PR DESCRIPTION
**Pull request by BurningAXE**

### CODE REVIEW

File: app/src/main/res/layout/fragment_splash.xml [hunk 1]
```diff
@@ -17,10 +17,12 @@
 
             <ImageView
                 android:id="@+id/circle"
-                android:layout_width="120dp"
-                android:layout_height="120dp"
-                android:layout_marginTop="24dp"
-                android:src="@drawable/splash_circle"
+                android:layout_width="80dp"
```
Comment (nkumarcc) [inline app/src/main/res/layout/fragment_splash.xml:20]: How safe is it to use specific dims (as opposed to relative dims) for responsive things? I've been playing it by ear when to hard code vs relative, wonder if you have any rules of thumb

**Review by nkumarcc — CHANGES_REQUESTED** [review]
Lgtm, can you add the screenshot?

File: app/src/main/res/layout/fragment_splash.xml (hunk 1 above)
Comment (BurningAXE) [reply app/src/main/res/layout/fragment_splash.xml:20]: Not really. However, relative distances with fixed sizes are more problematic.
ConstraintLayout does allow to use relative sizes but it requires some more work to set it up correctly. So, at least for now, I prefer to stick with the ScrollView which guarantees that you will be able to reach all elements on all screens/orientations.

**Comment by BurningAXE** [comment]
Screenshot_1586808095
Screenshot_1586808100

**Review by nkumarcc — APPROVED** [review]
Awesome!

File: app/src/main/res/layout/fragment_splash.xml (hunk 1 above)
Comment (filippkowalski) [reply app/src/main/res/layout/fragment_splash.xml:20]: You can use `app:layout_constraintDimensionRatio` to set specific ratio and get rid of hardcoded sizes

Comment (BurningAXE) [reply app/src/main/res/layout/fragment_splash.xml:20]: Hmm, nice! Will try it!

**Pull request merged**